tutorials/networking/high_level_multiplayer.rst

```diff
@@ -103,388 +101,280 @@ for full IPv6 support.
 Initializing the network
 ------------------------
 
-The object that controls networking in Godot is the same one that controls everything tree-related: :ref:`SceneTree <class_SceneTree>`.
-
-To initialize high-level networking, the SceneTree must be provided a NetworkedMultiplayerPeer object.
+High level networking in Godot is managed by the :ref:`SceneTree <class_SceneTree>`.
 
-To create that object, it first has to be initialized as a server or client.
+Every node has a ``multiplayer`` property, which is a reference to a default ``MultiplayerAPI`` instance.
 
-Initializing as a server, listening on the given port, with a given maximum number of peers:
+It is possible to create a new ``MultiplayerAPI`` object and assign it to a ``NodePath`` in the the scene tree,
+which will override ``multiplayer`` for the node at that path and all of it's children.
```

Comment:
```suggestion
which will override ``multiplayer`` for the node at that path and all of it's descendants.
```
Not limited to just children

Reply:
What is the difference? I don't believe I've come across this term in Godot before.

Reply:
"Children" would imply just the direct children, and not grandchildren, great-grandchildren, etc.

Reply:
Makes sense.